app/controllers/clients.js

```diff
@@ -5,6 +5,18 @@ const Conversations = require('../models/conversations');
 const Messages = require('../models/messages');
 const Users = require('../models/users');
 
+function average (arr) {
```

Comment:
Would either decide that this is controller-specific and add it as an underscored method to the controller, or put it in a lib somewhere.